generic_ent_adapter.go

```
@@ -1,10 +1,10 @@
+// Package main exposes the generic Ent adapter used to translate filter trees into Ent predicates.
 package main
 
 import (
 	"encoding/json"
 	"fmt"
 	"io/ioutil"
```

golangci-lint (2.11.4)

[error] SA1019: "io/ioutil" has been deprecated since Go 1.19: As of Go 1.16, the same functionality is now provided by package [io] or package [os], and those implementations should be preferred in new code. See the specific function documentation for details. 

(staticcheck)